files/chef-marketplace-cookbooks/chef-marketplace/templates/default/reckoner.rb.erb

```diff
@@ -1,16 +1,18 @@
 updater.driver = '<%= node['chef-marketplace']['platform'] == 'aws' ? 'ec2' : node['chef-marketplace']['platform'] %>'
 checker.driver = '<%= node['chef-marketplace']['role'] == 'aio' ? 'chef_server' : 'compliance' %>'
-license.count = <%= node['chef-marketplace']['license_count'] %>
+license.count = <%= count = node['chef-marketplace']['license_count'] %>
```

Comment:
Don't need the variable assignment here.

Also, I assume if the type == "fixed" then we'll just take the default license count of "5" and output that here, but no one will care? Should we eliminate outputting this line altogether if type == "fixed"?